scapy/layers/tls/basefields.py

```diff
@@ -178,3 +209,44 @@ def getfield(self, pkt, s):
             return ByteField.getfield(self, pkt, s)
         return s, None
 
+
+### SSLv2 fields
+
+class _SSLv2LengthField(_TLSLengthField):
+    def i2repr(self, pkt, x):
+        s = super(_SSLv2LengthField, self).i2repr(pkt, x)
+        if pkt.with_padding:
+            x |= 0x8000
+        #elif pkt.with_escape:      #XXX
```

Comment:
SSLv2 headers have an 'escape' bit (which has never been used to my knowledge). This is the proper way to parse and register it, but as long as there is no support for rebuilding it, these lines should be commented out. I made this more explicit in the latest commit.